src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -135,7 +135,7 @@ Multiple user-friendly projects aim to improve the experience of setting up a cl
 
 Below are a few projects which can help you install and control clients just with a few clicks:
 
-- [DappNode](https://docs.dappnode.io/get-started/installation/custom-hardware/installation/overview/) - DappNode doesn't come only with a machine from a vendor. The software, the actual node launcher and control center with many features can be used on arbitrary hardware.
+- [DappNode](https://docs.dappnode.io/user/quick-start/first-steps/) - DappNode doesn't come only with a machine from a vendor. The software, the actual node launcher and control center with many features can be used on arbitrary hardware.
```

Comment:
@eduadiez Feel free to suggest a different URL if it makes more sense in this context.

Reply:
This one seems more relevant to custom hardware: https://docs.dappnode.io/user/quick-start/Core/installation

Reply:
Yes, this link is perfect, thank you!!